docs/orientation-guide.md

```diff
@@ -71,7 +71,7 @@ _**Phase B**: Set up locally_
 
 1.  Set up TEAMMATES development environment on your computer.  
     Important:  
-    * Follow instructions to the letter. _Install the specified versions of the tool stack, not the latest versions._  
+    * [Follow instructions to the letter.](./settingUp.md) _Install the specified versions of the tool stack, not the latest versions._  
```

Comment:
This hyperlink is better at the main sentence, i.e. at "Set up TEAMMATES development environment". Also, you don't need the `./` prefix.